src/main/resources/META-INF/rewrite/jakarta-ee-10.yml

```diff
@@ -257,6 +258,19 @@ recipeList:
       ignoreDefinition: true
 ---
 type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.jakarta.DeprecatedCDIAPIsRemoved40
+displayName: Remove deprecated API's not supported in CDI4.0
+description: >
+  Deprecated APIs have been removed in CDI 4.0. This recipe removes and updates the corresponding deprecated methods.
+recipeList:
+  - org.openrewrite.java.migrate.jakarta.RemoveBeanIsNullable
+  - org.openrewrite.java.migrate.jakarta.UpdateAddAnnotatedType:
+      methodPattern: jakarta.enterprise.inject.spi.BeforeBeanDiscovery addAnnotatedType(jakarta.enterprise.inject.spi.AnnotatedType)
```

Comment:
`UpdateAddAnnotatedType` is only used once, yet the implementation seems to switch between two types of method patterns. Should it be used once more? Or can we simplify the implementation to only support a single case?

Reply:
@timtebeek I added the javax case as well .Should be there in the final commit.